scripts/ench_books.zs

```diff
@@ -0,0 +1,37 @@
+
+// Efficiency 5
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:dustRedstone> * 16)
+    .outputs(<minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 5 as short, id: 32 as short}]}))
+    .duration(100)
+    .EUt(14)
+    .buildAndRegister();
+
+// Unbreaking 3
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:ingotInvar> * 16)
+    .outputs(<minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 3 as short, id: 34 as short}]}))
+    .duration(100)
+    .EUt(14)
+    .buildAndRegister();
+
+// Unbreaking 4
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:ingotTitanium> * 16)
+    .outputs(<minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 4 as short, id: 34 as short}]}))
+    .duration(100)
+    .EUt(896)
+    .buildAndRegister();
+
+// Mending 1
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:ingotGold> * 64)
+    .inputs(<ore:ingotGold> * 16)
```

Comment:
These inputs can be made much more interesting. Instead of 80 gold ingots, I would suggest using 40 Rose Gold Plates, and 40 Sterling Silver Plates.